src/javascript/app/pages/user/account/settings/personal_details.js

```diff
@@ -129,6 +129,16 @@ const PersonalDetails = (() => {
         }
     };
 
+    const modifyContactCSMessage = (immutable_fields) => {
+        if (!/tax/.test(immutable_fields.toString())) {
+            $('.rowCustomerSupport').text((index, text) => text.replace(localize('To change your name, date of birth, country of residence, email, or tax information'),
+                localize('To change your name, date of birth, country of residence, or email')));
```

Comment:
I dont think this will cater the case for translated text, since we are checking the html text

Reply:
After the text is translated by translators, it's going to work because both arguments of the .replace() method are first computed and then applied. The main thing here to make it work is to ask translators to translate the parts of the Contact CS message, which I passed into localize(), in the exact same way they translated this parts inside the entire Contact CS message before.

Reply:
@maryia-deriv Does this replace only a part of the sentence? I'm not sure the way our translators translate but I think there might be cases when the order of the words is altered in other languages, so it won't match with a part of it.

Reply:
Perhaps, we can translate the entire sentences fully and just provide the relevant one beforehand, instead of modifying DOM on the fly.

Reply:
It would work well If translators simply copied & pasted their previous translations from the full initial string to the new parts of it, and it would be enough to make translation the same for the parts of the sentence in the first localize() argument of each .replace() method only. Translation of the second localize() argument wouldn't impact the result in this case.
However, I changed my PR the way you suggested by adding more options for the Contact CS string in order to not modify DOM on the fly.